climada/engine/unsequa/calc_cost_benefit.py

```diff
@@ -198,7 +198,7 @@ def uncertainty(self, unc_data, pool=None, **cost_benefit_kwargs):
         with log_level(level='ERROR', name_prefix='climada'):
             if pool:
                 LOGGER.info('Using %s CPUs.', pool.ncpus)
-                chunksize = min(unc_data.n_samples // pool.ncpus, 100)
+                chunksize = max(min(unc_data.n_samples // pool.ncpus, 100), 1)
```

Comment:
I am currently rewriting the parallel computing for the unsequa module as it was not working properly. So these changes are probably not needed. I will update at the end of today.

Reply:
Is being done in #763. I propose to merge #763 first.

Reply:
Actually, since #763 might take some time to review, we can also merge this pull request first.